src/v3/src/components/Button/Button.tsx

```diff
@@ -84,7 +84,7 @@ const Button: UISchemaElementComponent<{
       type={type}
       variant={variant ?? 'primary'}
       isFullWidth={wide ?? true}
-      buttonFocusRef={focusRef}
+      buttonRef={focusRef}
```

Comment:
Sorry about having to make these variable naming updates, Odyssey asked me to change the variable naming since it would only be a breaking change for us because we are the only team using this ref